ssl.conf

```diff
@@ -9,8 +9,9 @@ ssl_session_cache shared:SSL:50m;
 ssl_session_tickets off;
 
 # modern configuration. tweak to your needs.
-ssl_protocols TLSv1.2;
+ssl_protocols TLSv1.3 TLSv1.2;
```

Comment:
Do you know what happens if nginx doesn't support TLS1.3? (Which isn't that rare yet)